vue.config.js

```diff
@@ -93,6 +100,7 @@ module.exports = {
     },
     webpackBundleAnalyzer: {
       analyzerMode: 'disabled'
+      //generateStatsFile: true
```

Comment:
I would like to leave this comment here if it's okay with everybody else? This line means that a stats.json file gets generated, which is what I used at https://chrisbateman.github.io/webpack-visualizer/ to visualize the node-modules in our bundle. Might be useful again in the future?

Reply:
This is fine to leave in but please add a comment to the code about why this is commented out 👍🏻